src/clients/heroku.js

```diff
@@ -38,3 +37,27 @@ export function requestHerokuToken(email, password) {
       });
   });
 }
+
+export function createApp(name, apiKey) {
+  log.debug('createApp', name);
+  log.debug('createApp', apiKey);
```

Comment:
Are you sure we should be logging the API key here?

Reply:
removed